docs/DeveloperGuide.md

```diff
@@ -257,27 +257,49 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * *`  | user    | edit my student contacts                                     | My contact list is more extensive/flexible.minimize chance of someone else seeing them by accident |
+| `* * *`  | user    | delete my student contacts                                   | remove student’s contact of students that I don’t teach anymore.                                   |
+| `* * *`  | user    | use the help section                                         | learn the available commands in the application.                                                   |
+| `* * *`  | user    | import my data                                               | backup data and open on another device.                                                            |
+| `* * *`  | user    | export my data                                               | load data into a new device.                                                                       |
+| `* * *`  | user    | have a secure delete of my data                              | not accidentally delete information. (Need to type password to delete).                            |
+| `* * `   | user    | create progress report                                       | create tasks to keep track of the student’s progress.                                              |
+| `* * `   | user    | check the student’s taskings                                 | understand how good that student is doing.                                                         |
+| `* * `   | user    | edit the student’s tasking                                   | edit the incorrect data.                                                                           |
+| `* * `   | user    | delete / mark student’s tasking(s)                           | identify what taskings are done/obsolete.                                                          |
+| `* * `   | user    | see a calendar                                               | view on which day I have classes.                                                                  |
+| `* * `   | user    | extract students' progress report                            | show the parents their kids' performance.                                                          |
+| `* `     | user    | note down a more detailed class description                  | know what I need to do for a certain class.                                                        |
+|  `* `    | user    | filter calendar                                              | see clearly how many classes I have within a period of time (week/month, etc.)                     |
+|  `* `    | user    | wildcard search                                              | know what I can do on the app if I forgot the exact command I want to execute.                     |
```

Comment:
I want to be able to do a wildcard search
OR
I want to be able to get a recommendation of the commands that match with the words that I typed

I can be reminded of the exact command format if I forgot the exact command that I want to execute
OR
I can be reminded of the exact command format that I want to execute